src/server/worldserver/worldserver.conf.dist

```diff
@@ -2029,10 +2029,14 @@ Rate.Drop.Item.ReferencedAmount = 1
 #        Default:     1 - (Rate.XP.Kill, affects only kills outside of Battlegrounds)
 #                     1 - (Rate.XP.Quest)
 #                     1 - (Rate.XP.Explore)
-
-Rate.XP.Kill    = 1
-Rate.XP.Quest   = 1
-Rate.XP.Explore = 1
+#                     1 - (Rate.XP.Weekend)
+#                    66 - (Rate.XP.Weekend.EVID)
```

Comment:
Thinking on switch to http://www.boost.org/doc/libs/1_39_0/doc/html/date_time/gregorian.html to avoid the need of use event.